src/Select.tsx

```diff
@@ -16,15 +17,35 @@ export * from './generated/Select.js';
 
 export type SelectReactRendererProps = ReactSimpleRendererProps<SelectElement>;
 
+type SelectRenderer = ComponentType<SelectReactRendererProps>;
+
 export type SelectProps = Partial<Omit<_SelectProps, 'children' | 'renderer'>> &
   Readonly<{
-    children?: ReactNode | ComponentType<SelectReactRendererProps>;
-    renderer?: ComponentType<SelectReactRendererProps> | null;
+    children?: ReactNode | SelectRenderer | Array<ReactNode | SelectRenderer>;
+    renderer?: SelectRenderer | null;
   }>;
 
 function Select(props: SelectProps, ref: ForwardedRef<SelectElement>): ReactElement | null {
+  const children = Children.toArray(props.children as ReactNode[]);
```

Comment:
I see a couple of repeating code here and below, at line 40. I guess, it also could fail if children is a `function`.

Reply:
Updated to not use `Children.toArray` and only convert `props.children` to array here.